modules/apps/headless/headless-delivery/headless-delivery-test/src/testFunctional/macros/HeadlessWebcontentFolderAPI.macro

```diff
@@ -208,6 +208,20 @@ definition {
 		return ${curl};
 	}
 
+	macro _subscribeStructureContentFolder {
```

Comment:
Could you help update the name to _subscribe**Structured**ContentFolder? And maybe help go through the file to check all method names to see if any else need to be updated? Thanks